packaging/docker/kubernetes/Dockerfile

```diff
@@ -0,0 +1,84 @@
+# Dockerfile
```

Comment:
Aaron's been trying to consolidate this sort of stuff into https://github.com/FoundationDB/fdb-build-support

Not sure if this belongs here or there.  If you haven't yet, it probably makes sense to discuss it with him.